### PR TITLE
feat: add Routes page

### PR DESCRIPTION
**Pull request by nicolorevelli**
- Creation page
- Add route IPv4 and IPv6

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6777,6 +6777,7 @@
       "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
       "integrity": "sha512-l4Sp/DRseor9wL6EvV2+TuQn63dMkPjZ/sp9XkghTEbV9KlPS1xUsZ3u7/IQO4wxtcFB4bgpQPRcR3QCvezPcQ==",
       "dev": true
+
```
Comment (Tbaile) [inline package-lock.json:6780]: This bump in the keyboard is your "prettier" issue :laughing:

File: src/components/standalone/routes/RoutesContent.vue [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+              key: 'status',
+              label: t('standalone.routes.route_status_table')
+            },
+            {
+              key: 'actions'
+            }
+          ]"
+          :loading="loading"
+        >
+          <template #tbody>
```
Comment (andre8244) [inline src/components/standalone/routes/RoutesContent.vue:242]: This is fine, but that's the hard way: with NeTable you can specify a `template` for every table cell (like you did on the table below)

File: public/i18n/en/translation.json [hunk 3]
```diff
@@ -57,6 +57,13 @@
     "cannot_retrieve_wan_traffic": "Cannot retrieve WAN traffic",
     "register_unit_error": "Cannot register unit",
     "cancel_registration_error": "Cannot cancel registration",
+    "cannot_load_routes": "Cannot retrive routes",
+    "cannot_create_route": "Cannot create route",
+    "cannot_edit_route": "Cannot edit route",
+    "cannot_retrive_interfaces": "Cannot retrieve interfaces",
+    "cannot_retrive_route_types": "Cannot retrieve route types",
+    "invalid_metric": "Enter an integer number greater or equal than 0",
```
Comment (andre8244) [inline public/i18n/en/translation.json:65]:
```suggestion
    "invalid_metric": "Enter an integer number greater than or equal to 0",
```

File: public/i18n/en/translation.json [hunk 4]
```diff
@@ -57,6 +57,13 @@
...
+    "cannot_update_status_route": "Cannot update status route",
```
Comment (andre8244) [inline public/i18n/en/translation.json:66]:
```suggestion
    "cannot_update_status_route": "Cannot update route status",
```

**Review by andre8244 — APPROVED** [review]

**Pull request merged**